changelog.d/9542.bugfix

```diff
@@ -0,0 +1 @@
+Properly purge the event chain cover index when purging history.
```

Comment:
Should this be updated now that v1.29.0 has shipped with the initial fix?

Reply:
I think so